pyaedt/icepak.py

```diff
@@ -209,9 +217,22 @@ def create_source_blocks_from_list(self, list_powers, assign_material=True, defa
 
         Returns
         -------
-        type
+        list
             List of boundaries inserted.
 
+        Examples
+        --------
+
+        Create block boundaries from each box in the list. 
+
+        >>> box1 = icepak.modeler.primitives.create_box([1, 1, 1], [3, 3, 3], "BlockBox1", "copper")
+        >>> box1 = icepak.modeler.primitives.create_box([2, 2, 2], [4, 4, 4], "BlockBox2", "copper")
```

Comment:
`box2`

Reply:
@MaxJPRey yes, you're right. thanks for catching that.